index.html

```diff
@@ -0,0 +1,212 @@
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="flex project btn-flex">
+                    <li class="proj-btn">html</li>
+                    <li class="proj-btn">bootstrap</li>
+                    <li class="pproj-btn">Ruby</li>
+                 </ul>
+                </div>
+                <a href="/" class="cursor project-link">See Project</a>
+             </li>
+             <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+             </li>
+             <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="pproj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+            </li>
+            <li class="proj-con">
+              <div class="proj-txt">
+                <h3 class="proj-head">Professional Art Printing Data</h3>
+                <p class="proj-para">
+               A daily selection of privately personalized reads; no accounts or
+               sign-ups required. has been the industry's standard dummy text ever
+               since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+               <ul class="flex project btn-flex">
+                  <li class="proj-btn">html</li>
+                  <li class="proj-btn">bootstrap</li>
+                  <li class="proj-btn">Ruby</li>
+               </ul>
+              </div>
+              <a href="/" class="cursor project-link">See Project</a>
+           </li>
+            </ul>
+          </section>
```

Comment:
Kindly align this section properly
Yours:
![Screenshot (67)](https://user-images.githubusercontent.com/52497246/121771685-bf36a980-cb68-11eb-873a-8a2b1f23d056.png)

Original :
![Screenshot (64)](https://user-images.githubusercontent.com/52497246/121771697-d2497980-cb68-11eb-8632-dac6337da9a6.png)